doc/project/tsc.rst

```diff
@@ -0,0 +1,218 @@
+
+A representative’s suspension will end and voting rights be restored at the start
+of the next attended meeting. The TSC enforces the suspension policy for voting
+members who miss three consecutive TSC weekly meetings.
+
+Multi-day meetings (F2F events) are counted as "one" meeting. The TSC voted on
+February 16, 2022 to discontinue default enforcement of the suspension policy.
+The TSC voted on January 18, 2023 to re-enact default enforcement of the
+suspension policy.
+
+Notice of suspension will be sent to representatives who miss three consecutive
+meetings, noting that rights will be restored upon next attendance of a TSC
+meeting.
+
+.. note::
+
+   As per Section 4b of the Project Charter, Platinum and Silver
+   Members may choose to opt out of a voting seat on the TSC.
+
+   Members who opt out and then wish to reclaim their seat later will have their
+   voting rights restored at the start of the second consecutive meeting
+   attended following notification to the TSC Chair.
+
+
+Voting
+++++++
+
+Voting in the Zephyr Project is defined under Section 8 of the Project Charter.
+
+Additional points of clarity / TSC interpretation have been added below:
+
+- The Governing Board may opt to update the Charter to include the below
+  refinements. Until then, additional clarifications (if/where needed) will be
+  discussed in the Process Working Group, and approved in the TSC.
+- TSC In-Meeting Voting For items requesting an in-meeting vote of the Zephyr
+  Technical Steering Committee (TSC), assuming quorum requirements have been
+  met, the default voting mechanism will be a verbal motion to determine if
+  there is general consensus.
+- If there are no objections to a motion being brought forward, general
+  consensus is assumed and the motion passes.
+- Should there be any objections raised, the vote will move to email, and be
+  executed using the Voting Guidelines outlined in Section 8 of the Project
+  Charter.
+- Should a motion be deemed urgent by the TSC Chair, and assuming quorum
+  requirements have been met, the Chair may call for a roll call vote
+  in-meeting.
+
+Voting Options
+--------------
+
+- Voting Options are:
+
+  - "Yes",
+  - "No" or
+  - "Abstain"
+
+Abstention
+----------
+
+ - Abstentions do not count in tallying the vote negatively or positively; when
+   members abstain, they are in effect attending only to contribute to a quorum.
+ - Abstentions count toward Quorum (see present or participating
+   electronically below).
+ - Abstentions do not impact the number of votes needed to decide a vote
+
+Quorum
+------
+
+Quorum for TSC meetings shall require 60% of the voting representatives... (ref
+8b of the Charter)
+
+Decisions
+---------
+
+Decisions by vote shall be based on a majority vote, provided that at least
+sixty percent (60%) of the **TSC** representatives must be either
+present or participating electronically or by written action in order to
+conduct a valid vote. (ref 8c of the Charter)
```

Comment:
"provided that at least
sixty percent (60%) of the **TSC** representatives must be either
present or participating electronically or by written action in order to
conduct a valid vote. (ref 8c of the Charter)"

Redundant with Quorum definition above ?

Reply:
agree this is some repetition, but helps within the context.